text.css

```diff
@@ -0,0 +1,105 @@
+*{
+    margin: 0;
+}
+.container{
+    display: flex;
+    background-color: lightblue;
+    padding: 1rem;
+    box-shadow: 0px 3px whitesmoke;
+}
+#logo{
+    font-size: 55px;
+    margin-left: 45px;
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
+    padding: 0px 15px;
```

Comment:
you can store these spacing values variables and reuse them so that you have consistent spacing in your app, less chances of mistakes that way.